tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -43,21 +43,33 @@ object Discover {
   /** Find default clang compilation options. */
   def compileOptions(): Seq[String] = {
     val includes = {
-      val includedir =
-        Try(Process("llvm-config --includedir").lineStream_!.toSeq)
+      val llvmIncludeDir =
+        Try(Process("llvm-config --includedir").lineStream_!)
```

Comment:
The `.toSeq` must stay. Otherwise exceptions thrown during reading will not be caught by the `Try`.